project/gen2.scala

```diff
@@ -22,21 +24,26 @@ object gen2 {
     ""
   )
 
+  implicit val DurationMeta: Meta[Duration] =
+    Meta[Long].xmap(Duration.ofMillis, _.toMillis)
+
   object ToDeclaration extends Poly1 {
```

Comment:
I wanted a couple of `Duration` fields to be part of the generated class for Flamingos2 `ReadMode`.  Part of the process of generating the class is reading the `e_f2_read_mode` class where the data is stored as an integral type.  Since it needs to convert that into a `Duration` it has to be told how.  This is the same thing that is done over in the `dao` package as you noted above.  It's repeated here because it's not available otherwise.  Maybe it makes sense to take some of these mappings for common things like `Duration` and `Timestamp` and move it to somewhere that it can be shared, if possible.  I'm not sure where exactly though because this is needed for code generation before the rest of the code is built.